src/Avalonia.Controls/Presenters/TextPresenter.cs

```diff
@@ -288,6 +288,12 @@ public bool RevealPassword
             set => SetValue(CaretBrushProperty, value);
         }
 
+        public TimeSpan CaretBlinkInterval
```

Comment:
Missing comments which should always be added for new properties. They can be inherited from TextBox (or more likely TextBox inherits from here).

Reply:
I'm familiar with using <inheritdoc/> to inherit comments from base classes- but since TextBox just uses a TextPresenter and doesn't necessarily inherit from it, I'm not sure how this can be done. For now, I've copied the comment over.

Reply:
Something like this should work: https://github.com/AvaloniaUI/Avalonia/pull/13567#discussion_r1389514933.

Reply:
> Something like this should work: [#13567 (comment)](https://github.com/AvaloniaUI/Avalonia/pull/13567#discussion_r1389514933).

Did not know this was possible. Fixed it. Thank you.